integrations/spark/integrations/sparksql/1.json

```diff
@@ -58,6 +59,12 @@
             ],
             "child": 0
           },
+          {
+            "class": "org.apache.spark.sql.catalyst.plans.logical.Project",
+            "num-children": 1,
```

Comment:
Is there a standardized way we want to name fields or we don't naming control for logical plans?

Reply:
The logical plan is generated by Spark directly- we just call their `toJson` method.